src/betterbib/cli/doi2bibtex.py

```diff
@@ -18,20 +21,50 @@ def _get_version_text():
     )
 
 
+def _create_key_for_entry(entry):
+    with open(
+        os.path.dirname(os.path.realpath(__file__))
+        + "/nltk_english_stop_words.txt",
+        "r",
+    ) as f:
+        nltk_english_stop_words = set(f.readlines())
```

Comment:
Using stop words of nltk for slugifying the title.

Given the title `The Reproducibility Crisis`, instead of generating the slug `the_reproducibility_crisis` you'd get `reproducibility_crisis` as the `The` stopword was skipped